src/entry/setupRealtime.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This is used to get nudging/focus to work for students in Scratch JR.
+ */
+
+import { getDatabase } from 'firebase/database';
+import { initializeApp } from 'firebase/app';
+import { ref } from 'firebase/database';
+import { onChildChanged } from 'firebase/database';
+
+function getFirebaseRef ( userID ) {
+    // Returns reference to a node on Realtime Database"
+    var isLocal = window.isLocal;
+
+    var FIREBASE_URL = 'https://live-dashboard-d4d0e-default-rtdb.firebaseio.com/';
+    var refPath = isLocal ? 'LOCAL_NAV/' : 'LIVE_NAV/';
+    refPath += userID;
+    var firebaseApp = initializeApp({
```

Comment:
we might need to be careful here  - if firebase is already on the window this can cause issues. lets hold off for now and talk about it, maybe we can loop in @john-kelly as well